### PR TITLE
Add ability to merge one annotation store into another.

### PR DESCRIPTION
**Pull request by nickmshelley**

### CODE REVIEW

File: LDSAnnotations/AnnotationStore+Merge.swift [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        let notebooks = otherStore.notebooks()
+        for notebook in notebooks {
+            do {
+                try addNotebook(uniqueID: notebook.uniqueID, name: notebook.name, description: notebook.description, status: notebook.status, lastModified: notebook.lastModified, source: .Sync)
+            } catch {
+                NSLog("Unable to add notebook: \(error)")
+            }
+        }
+        
+        for annotation in otherStore.annotations() {
```
Comment (stephanheilner) [inline LDSAnnotations/AnnotationStore+Merge.swift]: Another option would be to use the function called `duplicateAnnotation`, refactor it to take an annotationStore and then you don't have to duplicate all this logic?

**Comment by nickmshelley** [comment]
Tests pass locally.

**Comment by stephanheilner** [comment]
Suggestion or 👍

File: LDSAnnotations/AnnotationStore+Merge.swift [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+//
+//  Created by Nick Shelley on 11/1/16.
+//  Copyright © 2016 Hilton Campbell. All rights reserved.
+//
+
+import Foundation
+
+extension AnnotationStore {
+    
+    public func mergeWith(otherStore: AnnotationStore, appSource: String, device: String) {
```
Comment (hiltonc) [inline LDSAnnotations/AnnotationStore+Merge.swift]: I think the name could be a little clearer. Maybe: addAnnotations(from otherStore: AnnotationStore, appSource: String, device: String)

Comment (nickmshelley) [inline LDSAnnotations/AnnotationStore+Merge.swift]: I like that better, except it also adds notebooks, and I don't really like `addAnnotationsAndNotebooks(from...)`. Maybe `addAll(from...)`? I dunno.

**Comment by nickmshelley** [comment]
Tests pass locally.

**Pull request merged**